### PR TITLE
Added some TODO for the collaborative project

### PR DESCRIPTION
**Pull request by alexandergmzx**
Hello, I did some TODO for the project

### CODE REVIEW

File: src/img.md [hunk 1]
```diff
@@ -0,0 +1 @@
+/*TODO: add an img fro the project*/
```
Comment (mocadev1) [inline src/img.md]: Hey @alexandergmzx here you have a typo it is "for" instead of "fro"

File: src/app.md [hunk 2]
```diff
@@ -0,0 +1 @@
+/*TODO: Set app headers*/
```
Comment (mocadev1) [inline src/app.md]: And what about the CORS?

**Review by mocadev1 — CHANGES_REQUESTED** [review]
Did some observations to be checked by @alexandergmzx

File: src/img.md (hunk 1 above)
Comment (alexandergmzx) [reply src/img.md]: Ok, this may be resolved in next iteration.

Comment (mocadev1) [reply src/img.md]: I see that you already did the changes, thank you.

File: src/app.md (hunk 2 above)
Comment (mocadev1) [reply src/app.md]: I that you already did the changes too, thank you.

**Review by alexandergmzx — COMMENTED** [review]
aaa

**Pull request merged**